packages/privacy-toolset/src/do-not-sell-dialog/styles.scss

```diff
@@ -0,0 +1,122 @@
+@use "sass:math";
+@import "../styles/fonts.scss";
+
+.do-not-sell,
+.do-not-sell * {
+	font-family: $default-font-family;
+	box-sizing: border-box;
+}
+
+.do-not-sell {
+	display: flex;
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	z-index: 50002;
+}
+
+.do-not-sell__dialog {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	margin: auto;
+	width: 100%;
+	max-width: 700px;
+	height: auto;
+	max-height: 90vh;
+	background-color: #fff;
+	z-index: 1;
+	overflow-y: auto;
+	box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2), 0 1px 3px rgba(0, 0, 0, 0.15), 0 1px 0 rgba(0, 0, 0, 0.05);
+	border-radius: 4px;
+}
+
+.do-not-sell__header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	border-bottom: 1px solid #e0e0e0;
+	padding: 1rem 2rem;
+}
+
+.do-not-sell__close-button {
+	background: none;
+	border: none;
+	margin: 0;
+	padding: 0;
+	cursor: pointer;
+}
+
+.do-not-sell__close-button::after {
+	display: inline-block;
+	content: "\00d7";
+	font-size: 1.75rem;
+	transform: translateX(1px);
```

Comment:
No, transforming the icon is to move it to a better position visually. This is how it looks without transform:
<img width="409" alt="CleanShot 2022-12-08 at 10 54 51@2x" src="https://user-images.githubusercontent.com/8419292/206514807-c6fefb8a-f4e7-48d2-9015-c3ba6f29f638.png">

And with transform:
<img width="410" alt="CleanShot 2022-12-08 at 10 56 09@2x" src="https://user-images.githubusercontent.com/8419292/206515115-76a982cf-cb3b-4c21-9698-a62c6cf7979d.png">

Reply:
I've added a padding-left to the icon though so to ensure it will break quicker on smaller screens, preventing borked design as you attached. Thanks for the suggestion!